lms/static/sass/elements/_banners.scss

```diff
@@ -44,3 +44,50 @@ $full-width-banner-margin: 20px;
         right: $full-width-banner-margin;
     }
 }
+
+.page-banner {
+    max-width: $lms-max-width;
+    margin: 0 auto;
+
+    .user-messages {
+        padding-top: $baseline;
+    }
+
+    .alert {
+        margin-bottom: $baseline !important;
```

Comment:
I didn't like the !important either but I couldn't find a way to get this to work otherwise. This is for v1 Sass (aka legacy) not for Bootstrap, and there is a rule that seems to win no matter how specific this rule is.